Resources/Prototypes/Entities/Structures/Machines/lathe.yml

```diff
@@ -554,6 +554,13 @@
       - MagazineBoxMagnumIncendiary
       - MagazineBoxLightRifleIncendiary
       - MagazineBoxRifleIncendiary
+      - ShellShotgunPractice
+      - MagazineBoxPistolPractice
+      - MagazineBoxMagnumPractice
+      - MagazineBoxLightRiflePractice
+      - MagazineBoxRiflePractice
+      - WeaponLaserCarbinePractice
+      - WeaponDisablerPractice
       - Signaller
       - SignalTrigger
       - VoiceTrigger
```

Comment:
Can you alphabetise all these recipes please?

Reply:
How so exactly? Just the practice section or for the entire list?

Reply:
Entire list is preferable as people will just copy whatever formatting is there most of the time.

Reply:
Fair enough, fixed!